mesecons/util.lua

```diff
@@ -403,6 +403,7 @@ function mesecon.vm_get_node(pos)
 end
 
 -- Sets a node’s name during a VoxelManipulator-based transaction.
+-- Aliases may not work correctly.
```

Comment:
```suggestion
-- Aliases may not work correctly if mesecon.get_conductor(name).states was touched by a mod.
```

That's how I understood the problem, right? If you want to make sure they're not modified, consider metatables. At least the reason *why* it may not work correct should be stated here.

Reply:
The issue can occur if a mod swaps in a node whose name is actually an alias.

Reply:
Then please be so nice to mention in which cases this function is expected to work, and when not.

Reply:
I will make some changes after #614 is merged or closed. If it is merged, I can make some further changes to the code then remove the notice. This PR is basically blocked for now.